docs/configuration/holmesgpt/toolsets/grafanatempo.rst

```diff
@@ -18,12 +18,53 @@ Check out this `video <https://www.loom.com/share/f969ab3af509444693802254ab0407
 Getting Grafana URL
 -----------------------
 
-You can find the Grafana URL required for Loki and Tempo in your Grafana cloud account settings. 
+You can find the Grafana URL required for Tempo in your Grafana cloud account settings.
 
 .. image:: /images/grafana-url-for-holmes.png
   :width: 600
   :align: center
 
+
+Obtaining the datasource UID
```

Comment:
Consider to move it to another md file and load here since we are doing the same in multiple places.